reactor-core/src/main/java/reactor/core/publisher/UnicastProcessor.java

```diff
@@ -280,8 +328,7 @@ public Emission emitNext(T t) {
 					ex.initCause(e);
 				}
 			}
-			Operators.onDiscard(t, ctx);
-			onError(ex);
+			onError(ex); //FIXME should this really error ??
```

Comment:
this one is indeed inconsistent with other usages. `tryEmitNext` should probably not emit any error here

Reply:
I opened a separate issue, `SerializedManySink` is also guilty of a similar thing